src/lib/dux/user/actionTypes.ts

```diff
@@ -0,0 +1,22 @@
+import { CreateAction } from '../../../utils/typeHelpers/reducers/CreateAction';
+
+export const INIT_USER = 'INIT_USER';
+export const RESET_USER = 'RESET_USER';
+export const SET_USER_LOADING = 'SET_USER_LOADING';
+export const UPDATE_USER_INFO = 'UPDATE_USER_INFO';
```

Comment:
lets remove this and use `USER_ACTIONS .xxx`

Reply:
Oh I missed to remove them, thank you